jekyll/_cci2/configuration-reference.adoc

```diff
@@ -2653,6 +2653,51 @@ A job can have the keys `requires`, `name`, `context`, `type`, and `filters`.
 
 A job name that exists in your `config.yml`.
 
+'''
+====== `serial-group`
```

Comment:
I think it's worth mentioning somewhere that serial groups will wait for a maximum of 5 hours before being cancelled.

This only applies to groups waiting in the queue, the standard limits apply to the jobs inside the queue (e.g. max run time of build jobs).

Reply:
What does "cancelled" mean in this scenario? Like the serial group cancels so all the waiting jobs will just run concurrently after 5 hours?